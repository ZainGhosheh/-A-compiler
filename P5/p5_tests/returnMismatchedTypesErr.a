f : () -> bool {
	a:int;
	a = 4;
    return a;
}
