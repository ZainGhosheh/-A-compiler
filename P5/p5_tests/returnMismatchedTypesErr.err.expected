FATAL [4,5]-[4,13]: Bad return value
Type Analysis Failed
